index.html

```diff
@@ -5,12 +5,33 @@
         <title>ubio application bundle</title>
         <link href="/src/css/index.css" rel="stylesheet">
         <script type="module">
-            window.location.hash = '';
-            localStorage.clear();
-
             import { createEndUserSdk } from '/web_modules/@ubio/sdk.js';
             import { createApp } from '/src/main.js';
 
+            async function getExisting() {
+                const existingJob = localStorage.getItem(`job-session`);
+
+                if (!existingJob) {
+                    return null;
+                }
+
+                try {
+                    const { token, jobId, serviceId, supportEmail, local, domain } = JSON.parse(existingJob);
+                    const sdk = createEndUserSdk({ token, jobId, serviceId });
+                    const job = await sdk.getJob();
+
+                    if (job.status === 'success' || job.status === 'fail') {
```

Comment:
Isn't `job.finishedAt` more future proof?

Reply:
Good idea :+1:

Reply:
This is also a good idea for SDK. Job tracking automatically stopped for end states.